AB_environments/README.md

```diff
@@ -154,6 +154,9 @@ categories:
   - runtime
   - benchmarks
   - stability
+max_parallel:
```

Comment:
The same way we explain opther parameters (e.g category, test_null_hypothesis, etc ) in the README, can we explain these two in the README as well? 

I know there are some comments in the `config.yml` but it'll be useful having them explain in the readme too.

Reply:
done